.github/workflows/deploy-scripts-internally.yaml

```diff
@@ -34,7 +34,14 @@ jobs:
 
                     cd /tmp/nango-temp/nango-integrations
                     npm install zod soap botbuilder
-                    nango admin:deploy-internal template-$integration
+
+                    response=$(nango admin:deploy-internal template-$integration)
+
+                    echo "$response" | jq '.error.code' | grep -q "unknown_provider_config"
+                    if [ $? -eq 0 ]; then
+                      echo "Error: Unknown provider config for $integration"
+                      return true
```

Comment:
does this return mean the script will exist at the first missing integration? is that what we want?

Reply:
not at all! Thanks, updated